src/wled2.ts

```diff
@@ -48,13 +47,102 @@ export = (RED: Red): void => {
     });
   });
 
+  // This function will 
+  function which(
+    incomingPayloadValue: string | number | boolean | IWledSegment[],
+    nodePayloadValue: string | number | boolean | IWledSegment[],
+    wrapper?: NumberConstructor // Add more types if used for others later.
+  ): string | number | boolean | IWledSegment[] {
+    const val = incomingPayloadValue ?? nodePayloadValue ?? undefined;
+    return wrapper && val !== undefined ? wrapper(val) : val
+  }
+
+  function getMergedPayloadAndConfig(payload: IWledNodeProperties, config: IWledNodeProperties): IWledNodeProperties {
+    const mergedProperties = {
+      name: "N/A"
+    } as IWledNodeProperties
+    mergedProperties.address = which(payload.address, config.address) as string

```

Comment:
This can just be `payload.address ?? config.address` can't it? Does it even need an extra `?? undefined` at the end?

Reply:
true